### PR TITLE
Update clear, and tests

### PR DESCRIPTION
**Pull request by imurchie**
Make use of `adb.clearTextField` to try to clear better in Android.

### CODE REVIEW

File: test/functional/commands/keyboard-e2e-specs.js [hunk 1]
```diff
@@ -42,6 +44,12 @@ async function runTextEditTest (driver, testText, keys = false) {
   return el;
 }
 
+async function clearKeyEvents (driver) {
+  let el = _.last(await driver.findElOrEls('class name', 'android.widget.Button', true));
+  driver.click(el.ELEMENT);
+  await B.delay(500);
```
Comment (jlipps) [inline test/functional/commands/keyboard-e2e-specs.js]: maybe a comment about the delay?

File: test/functional/commands/keyboard-e2e-specs.js [hunk 2]
```diff
@@ -42,6 +44,12 @@ async function runTextEditTest (driver, testText, keys = false) {
   return el;
 }
 
+async function clearKeyEvents (driver) {
```
Comment (jlipps) [inline test/functional/commands/keyboard-e2e-specs.js]: sorry I'm a little confused what this function name means. How does this function clear keyevents? Or is it sending "clear key" events? Maybe a note about what button this is tapping and how that is related might help?

File: test/functional/commands/keyboard-e2e-specs.js [hunk 3]
```diff
@@ -83,22 +96,21 @@ let tests = [
...
 
 let languageTests = [
   { label: 'should be able to send Tamil', text: 'சோதனை' },
   { label: 'should be able to send Gujarati', text: 'પરીક્ષણ' },
   { label: 'should be able to send Chinese', text: '测试' },
   { label: 'should be able to send Russian', text: 'тестирование' },
-  // skip rtl languages, which don't clear correctly atm
-  // { label: 'should be able to send Arabic', 'تجريب'],
-  // { label: 'should be able to send Hebrew', 'בדיקות'],
+  { label: 'should be able to send Arabic', text: 'تجريب' },
```
Comment (jlipps) [inline test/functional/commands/keyboard-e2e-specs.js]: yay!

File: test/functional/commands/keyboard-e2e-specs.js (hunk 2 above)
Comment (imurchie) [inline test/functional/commands/keyboard-e2e-specs.js]: Will do. Essentially it just clears the view (the view just takes any key events coming in and prints their details out in a TextView). Previously we were clearing it by reloading the activity, but that is both inefficient and potentially hides issues with how the events are being processed (less so for key events, but I removed it in the interest of consistency).

Comment (jlipps) [inline test/functional/commands/keyboard-e2e-specs.js]: ahhh ok

**Pull request merged**